setup.py

```diff
@@ -31,6 +31,7 @@
     "spacy.vocab",
     "spacy.attrs",
     "spacy.kb",
+    "spacy.kb_inmemorylookup",
```

Comment:
Maybe a bit too long to parse, how about `kb_inmemory_lookup`?

Reply:
Why not `kb_in_memory_lookup`?

Reply:
Yea, that's better yet.

Reply:
Since we'll have a `kb` subdir, should we rename the modules to smth. like `base.pyx` and `in_memory_lookup.pyx`? Or do we want to stick with `kb.pyx` and `kb_in_memory_lookup.pyx`?

Reply:
I'd lean to `kb/kb.pyx` and `kb/in_memory.pyx`, maybe the "lookup" bit is not needed for the module name? We're not going to create another in-memory implementation any time soon, right?

Reply:
Don't think so, no. Adjusted accordingly.